package.json

```diff
@@ -53,6 +55,11 @@
     "tabWidth": 4
   },
   "devDependencies": {
+    "@storybook/addon-a11y": "^3.4.8",
```

Comment:
can we have @storybook/cli here as well? that way we don't need a global install for it